hathor/p2p/node_sync.py

```diff
@@ -223,6 +228,10 @@ def __init__(self, protocol: 'HathorProtocol', downloader: Downloader, reactor:
         # Indicate whether the sync manager has been started.
         self._started: bool = False
 
+        # Indicate whether the synchronization is enabled.
+        # When the sync is disabled, it will keep the last synced_timestamp.
```

Comment:
How will the explorer handle this? As if they are out of sync? Are we going to trigger the alerts we have nowadays for important nodes out of sync?

Reply:
I think we will going to trigger alerts for nodes out of sync, because the alerts are based on the `synced_timestamp` property of this class, which will not be updated anymore when the sync is disabled.

This can be seen in https://github.com/HathorNetwork/hathor-p2p-monitor/blob/main/hathor_p2p_monitor/p2p_monitor_important_peers/p2p_monitor_peer_id.py#L91

Maybe we will need a patch there to check the new `is_enabled` property on the list of connected_peers, and don't alert for the ones which have the sync disabled.

Reply:
Should we add them to the `always_enable_sync` list?

Reply:
Answering @pedroferreira1 's question: The explorer will show as not synced. We can use the new `is_enabled` to show that we are not syncing with that node at that time.

Reply:
> Should we add them to the always_enable_sync list?

I don't think so, the list is not small.

I agree that we should patch both the p2p alert monitor and explorer to consider the `is_enabled` property now.